robust_pdf_plot.ipynb

```diff
@@ -0,0 +1,367 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #63.        # Genrate Samples from normal distribution
</pre><p>Fix Typo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Fixed this

Reply:
Fixed